ONSdigital/eq-author
# Update Front-end Build

### Changes

- Updated npm dependencies
- Adopts [fractal project structure](https://github.com/davezuko/react-redux-starter-kit/wiki/Fractal-Project-Structure)
- Adds [Redux CLI](https://github.com/SpencerCDixon/redux-cli)
- Removed all existing React components/tests/etc.

### How to test

- Remove existing node modules `rm -rf node_modules`
- `npm install`
- `npm run dev` to start dev server
- visit `localhost:3000` and confirm basic welcome message
- `npm run compile` and confirm app builds with no errors
- `npm run test` and confirm task passes (there are no tests)

### Who

Anyone
